developers/wallets/native/index.md

```diff
@@ -119,23 +124,70 @@ The `request` value is URL encoded. Its contents look like this when unencoded:
+
+To retrieve the credential via VC-API, the wallet first sends a `POST` request
+to the URL (extracted from `protocols.vcapi` above) with an empty JSON object
+(`{}`) in the body of the request to attempt to download the VC to be stored.
+
+```http
+POST /exchangers/z1A1GqykGBWKbwhFCDqFjMfnG/exchanges/z19mxa763DAKX7diL51kBFecZ
+Host: exchanger.example.com
+Content-Type: application/json
 
-See the [VC API Specification](https://w3c-ccg.github.io/vc-api/) for more details on this protocol.
+{}
+```
+
+The response to that request will contain a Verifiable Credential within a
+`verifiablePresentation` object. Additionally, if there are further steps
+required in the flow such as DID Auth a `verifiablePresentationRequest` object
+will describe the details.
 
-Example VCAPI protocol URL:
-`https://exchanger.example.com/exchangers/z1A1GqykGBWKbwhFCDqFjMfnG/exchanges/z19mxa763DAKX7diL51kBFecZ`
+Below is a partial example response:
+
+```json
+{
+  "verifiablePresentation": {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://www.w3.org/2018/credentials/examples/v1",
+      "https://w3id.org/security/suites/ed25519-2020/v1"
+    ],
+    "type": ["VerifiablePresentation"],
+    "holder": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "verifiableCredential": [{
+          "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://www.w3.org/2018/credentials/examples/v1",
+        "https://w3id.org/security/suites/ed25519-2020/v1"
+      ],
+      "id": "http://example.edu/credentials/3732",
+      "type": [
+        "VerifiableCredential",
+        "UniversityDegreeCredential"
+      ],
+      ...
+    }]
+  }
+}
+```
+
+More complete examples can be found in the
+[Example Exchanges](https://exchanger.example.com/exchangers/z1A1GqykGBWKbwhFCDqFjMfnG/exchanges/z19mxa763DAKX7diL51kBFecZ)
+section of the [VC-API Specification](https://w3c-ccg.github.io/vc-api/).
 
 ### OID4VCI
 
 The `OID4VCI` property will be a URL that includes all of the required information to complete the issuance of the Verifiable Credential using the OID4VCI protocol.
 
 Example OID4VCI protocol URL:
+<pre><code style="white-space: normal;overflow: auto;word-break: break-all;">
 `openid-credential-offer://?credential_offer=%7B%22credential_issuer%22%3A%22https%3A%2F%2Fexchanger.example.com%2Fexchangers%2Fz1A1GqykGBWKbwhFCDqFjMfnG%2Fexchanges%2Fz1A36rr6wEL25EEiikKvisVEC%22%2C%22credentials%22%3A%5B%7B%22format%22%3A%22ldp_vc%22%2C%22credential_definition%22%3A%7B%22%40context%22%3A%5B%22https%3A%2F%2Fwww.w3.org%2F2018%2Fcredentials%2Fv1%22%2C%22https%3A%2F%2Fpurl.imsglobal.org%2Fspec%2Fob%2Fv3p0%2Fcontext.json%22%5D%2C%22type%22%3A%5B%22VerifiableCredential%22%2C%22OpenBadgeCredential%22%5D%7D%7D%5D%2C%22grants%22%3A%7B%22urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Apre-authorized_code%22%3A%7B%22pre-authorized_code%22%3A%220065a8a0-069b-46f1-a857-4e1ce5047afd%22%7D%7D%7D`
```

Comment:
The pr preview looked wrong here. Is this ok live?  Either way, looks like extra backticks around the URL.

Reply:
My current Markdown highlighter in VIM annoyingly hides certain markup when you're not on the line. Makes it legible...but insane to clean/understand. I need to sort out which thing is doing that...and disable it.

Thanks for the catch!